test/extended/util/test.go

```diff
@@ -206,6 +206,12 @@ func createTestingNS(baseName string, c kclientset.Interface, labels map[string]
 		return fn(baseName, c, labels)
 	}
 
+	// if we're running an upstream test, make sure to skip openshift admission.
+	// TODO we may be able to relax this after the openshift apiserver is back in the openshift/installer
```

Comment:
bz to track that we dont forget to come back to this?

Reply:
> bz to track that we dont forget to come back to this?

I'm actually not sure. do we want to?